Dockerfile

```diff
@@ -22,9 +22,21 @@ RUN apt-get update \
 	&& rm -rf /var/lib/apt/lists/* \
 	&& docker-php-ext-install curl json mysql mysqli
 
+# Use the production php.ini unless PHP_DEBUG=1 (defaults to 0)
+ARG PHP_DEBUG=0
+RUN [ "$PHP_DEBUG" = "1" ] || \
+	{ echo "Using production php.ini" \
+		&& mkdir /usr/src/php \
+		&& tar --file /usr/src/php.tar.xz --extract --strip-components=1 --directory /usr/src/php \
```

Comment:
Where does `/usr/src/php.tar.xz` come from? Is it there in the base image as I can't see it being added

Reply:
Yeah, every php image has a tar'd copy of the source code in that location.